### ISSUE TITLE
Could You Provide a Preprocessed Dataset Example?

### ISSUE THREAD

**Issue by RichardMLuu** [huang-he99/STF#3]
Thanks for sharing your great work!
I am trying to process the dataset following the provided steps, but I find it quite cumbersome as it requires multiple steps. My dataset setup is as follows:
```
train_dataset = dataset_cls_func(
    dataset_name="STIL",
    data_root="data/spatio_temporal_fusion/Mini_STIF_Dataset_split_size_256_stride_256/STIL_Train",
    transform_func_list=train_transform_list,
)
```
Would it be possible to provide a fully preprocessed dataset like `data/spatio_temporal_fusion/Mini_STIF_Dataset_split_size_256_stride_256/STIL_Train`? This would allow users to directly load and use the dataset, significantly improving usability and saving a lot of time.
Thanks in advance for your help!

**Comment by huang-he99** [comment]
Thank you for your interest in our work!  
We understand that processing the dataset involves multiple steps, and we appreciate your feedback. However, due to the large data volume, providing a fully preprocessed dataset is not feasible. The current processing steps, including normalization, cropping to a multiple of 256, and splitting into smaller patches based on `patch_size` and `stride`, are designed to be straightforward and flexible. We encourage you to implement these steps based on your specific needs, as this approach allows for greater customization and adaptability to different use cases.  
We appreciate your understanding and hope this helps! Let us know if you have any further questions.
